.github/workflows/cimain.yaml

```diff
@@ -0,0 +1,189 @@
+name: Continuous Integration
+
+on:
+  push:
+    branches: [main]
+
+# See https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#example-using-concurrency-to-cancel-any-in-progress-job-or-run
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  runner:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 120
+    # We use the same job template, but parametrize the actual command to be passed to the runner
+    # binary using the matrix strategy, so that we get the commands running in parallel.
+    strategy:
+      fail-fast: false
+      matrix:
+        cmd:
+          # Do not add `run-ci` to these commands. We parse them for `runner run-ci`.
+          - check-format --scope=all
```

Comment:
Isn't scope=all the default now?

Reply:
Currently, the default is `--scope=diff-to-main`. I think when running locally we'd want to run the faster version by default. But I can change the default.